{{cookiecutter.app_name}}/.env.example

```diff
@@ -6,3 +6,7 @@ DATABASE_URL=sqlite:////tmp/dev.db
 GUNICORN_WORKERS=1
 LOG_LEVEL=debug
 SECRET_KEY=not-so-secret
+{% if cookiecutter.use_google_logging == "yes" %}
+GOOGLE_CLIENT_ID=GOOGLE_CLIENT_ID
+GOOGLE_CLIENT_SECRET=GOOGLE_CLIENT_ID
```

Comment:
These should also be added to [`settings.py`](https://github.com/cookiecutter-flask/cookiecutter-flask/blob/master/%7B%7Bcookiecutter.app_name%7D%7D/%7B%7Bcookiecutter.app_name%7D%7D/settings.py)

Reply:
They were there :) I've deleted unnecessary empty line there in fixup commit.